src/app/components/Times.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+// tslint:disable:max-line-length
+// tslint:disable-next-line:variable-name
+const SvgComponent = (props: React.HTMLProps<SVGElement>) => (
+  <svg width='15px' height='15px' viewBox='0 0 15 15' version='1.1' {...props}>
```

Comment:
Could the font-awesome icon be used for this?

Reply:
you'd think, but devika didn't like the font awesome times icon, so she made this one, the lines are thinner i think